package.json

```diff
@@ -26,6 +26,7 @@
     "react-scripts": "3.4.3",
     "reactjs-popup": "2.0.3",
     "redux": "4.0.5",
+    "redux-thunk": "^2.3.0",
```

Comment:
do we really need thunk? I'm trying to keep dependencies as few as possible

Reply:
I needed it because as far as I know, we can't do async dispatch with vanilla redux 🙂 it was the easiest way I know
If you know another way please tell me 🙂

Reply:
I needed it to call `initAuthentication` and `loadJDL` async dispatcher

Reply:
Ok let me take a look. I was planning to handle the async part within the action. But ya that might not work anyway I'll test this